website/versioned_docs/version-0.63/native-modules-setup.md

```diff
@@ -399,12 +399,12 @@ Next you need to create a YAML file for GitHub Actions, the basic steps are:
         node-version: '12.9.1'
 
     - name: Setup MSBuild
-      uses: microsoft/setup-msbuild@v1.0.0
+      uses: microsoft/setup-msbuild@v1
       with:
         vs-version: 16.5
        
     - name: Setup NuGet
-      uses: NuGet/setup-nuget@v1.0.2
+      uses: NuGet/setup-nuget@v1

```

Comment:
do we need to update 0.64 and next?

Reply:
It doesn't look like we have versioned docs for 0.64 yet. If the above is true and we are using msbuild's built-in nuget support then the change I made in next should be removed.